openmdao/docs/Makefile

```diff
@@ -41,7 +41,7 @@ clean:
 	rm -rf _srcdocs/*
 	rm -rf tags
 	rm -rf tmp
-	rm -f feature_docs_unit_test_db.sqlite
+	# rm -f feature_docs_unit_test_db.sqlite
```

Comment:
was commenting out the db a debugging thing or is it intentional?

Reply:
That's from the old numpydocs stuff that used to search through all the tests and find the ones we marked as features. I will take Steve's suggestion and delete the lines and code associated with that.